src/app/users/associateRoles.js

```diff
@@ -26,7 +25,7 @@ const getViewModel = async (req) => {
   const userOrganisations = userId.startsWith('inv-') ? await getInvitationOrganisations(userId.substr(4), req.id) : await getUserOrganisations(userId, req.id);
   const organisationDetails = userOrganisations.find(x => x.organisation.id === req.params.orgId);
   const policyEngineResult = await policyEngine.getPolicyApplicationResultsForUser(userId.startsWith('inv-') ? undefined : userId, req.params.orgId, req.params.sid, req.id);
-  const serviceRoles = policyEngineResult.rolesAvailableToUser;
+  const serviceRoles = await policyEngineResult.rolesAvailableToUser;
```

Comment:
Is `await` necessary here? Looking at the source code in policy engine this seems to be a vanilla array.